tests/infrastructure/replaying-projections.tests.coffee

```diff
@@ -65,7 +64,7 @@ describe 'Space.eventSourcing - replaying projections', ->
       SecondCollection.remove {}
       @event = new TestEvent sourceId: 'test123', value: 'test'
       @app = new TestApp()
-      @app.configure { appId: 'TestApp' }
+      @app.reset()
```

Comment:
Nice. Will be good to remove the appId from the Configuration as it should be set in the constructor if at all. There's an id of the running meteor process we can utilise to for abstract level identification.